packages/rules-core/fixtures/rationale/valid-rationale-truncated.json

```diff
@@ -0,0 +1,18 @@
+{
+    "foo": {
+        "bar": {
+            "baz": {
+                "foo": "foo",
+                "bar": "bar",
+                "baz": "baz"
+            }
+        }
+    },
+    "rationale": {
+        "foo.bar.baz": {
+            "foo": "foo",
+            "bar": "bar",
+            "baz": "baz"
```

Comment:
Minor point, but it may be worthwhile to have values be different from keys just to make sure code or future changes never swap them somehow